src/OrchardCore.Modules/OrchardCore.Deployment/Views/Items/RecipeFileDeploymentStep.Fields.Edit.cshtml

```diff
@@ -4,7 +4,7 @@
 
 <div class="form-group">
     <div class="row">
-        <div class="col-sm">
+        <div class="all">
```

Comment:
Just I have a doubt about this one, or the intent to take the full width? If is that, why I didn't see `all` in SCSS file?

Also we could have a better meaningful name

Reply:
yeah good catch, typo

Reply:
It happens ;)